Common/SensorFusion/Inc/SensorFusion.hpp

```diff
@@ -131,5 +133,6 @@ Gps_Data_t SF_GetRawGPS();
  * @return Altimeter struct.
  */ 
 Altimeter_Data_t SF_GetRawAltimeter();
+#endif
```

Comment:
I feel like i'm getting repetitive with these `#endif`s... I'm just going to leave it be now even though there are more